packages/core/package.json

```
@@ -0,0 +1,62 @@
+{
+  "name": "@syntest/core",
+  "version": "0.2.0",
+  "description": "The common core of the SynTest Framework",
+  "keywords": [
+    "syntest",
+    "automatic-test-generation"
+  ],
+  "homepage": "https://www.syntest.org",
+  "bugs": {
+    "url": "https://github.com/syntest-framework/syntest-core/issues"
+  },
+  "license": "Apache-2.0",
+  "contributors": [
+    "Annibale Panichella",
+    "Mitchell Olsthoorn",
+    "Dimitri Stallenberg"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "/dist",
+    "/NOTICE"
```

The notice file is most likely in the root and not in this folder anymore

```suggestion
    "../../NOTICE"
```

---

Fixed.